rhobs/alerting/data_plane/prometheus.gitops_sync_alerts.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-gitops-sync-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: gitops_sync_alerts
+    interval: 1m
+    rules:
+    - alert: GitOpsApplicationSyncErrors
+      expr: |
+        (
+          sum(avg_over_time(argocd_app_info{sync_status="Synced", namespace="gitops-service-argocd"}[5m]))
+          or
+          sum(argocd_app_info{namespace="gitops-service-argocd"}) * 0
+        )
+        /
+        (
+          sum(avg_over_time(argocd_app_info{namespace="gitops-service-argocd"}[5m]))
+          or
+          sum(argocd_app_info{namespace="gitops-service-argocd"}) * 0
```

Comment:
I suppose this expression is just adding clutter and only results with dividing by zero. I think we can remove it.

Reply:
Removed.